src/Symfony/Bridge/Doctrine/ContainerAwareEventManager.php

```diff
@@ -46,7 +46,7 @@ public function __construct(ContainerInterface $container)
      *
      * @return bool
      */
-    public function dispatchEvent($eventName, EventArgs $eventArgs = null)
+    public function dispatchEvent(string $eventName, EventArgs $eventArgs = null)
```

Comment:
this is interesting, unfortunately we won't be able to enable this rule, as this we must keep BC and we'll still maintain 5.5 code for some years to come (3.4 being merged into master)

Reply:
What about adding parent vendors to skip? Something like this:

```yaml
RemoveUselessDocBlockFixer:
    exclude_parent:
         - 'Doctrine/*'
```